pkg/controller/clusterautoscaler/clusterautoscaler_test.go

```diff
@@ -125,3 +132,120 @@ func TestAutoscalerArgs(t *testing.T) {
 		}
 	}
 }
+
+type MockReconciler struct {
+	getAutoscalerOk bool
+	gASerrType      error
+	configVersion   string
+	isDCCResult     bool
+	calledIsDCC     bool
```

Comment:
What is `DCC`? Is this `isCurrent`?

Reply:
There's a function named "isDeploymentControllerCurrent"

So this variable is calledIsDCC, eg, did you call isDeploymentControllerCurrent?

Reply:
I read the two variables `isDCCResult` and `calledisDCC` and couldn't really grok what they were.

For `calledIsDCC` how about `isDCCFuncCalled`? Then it code it (possibly) reads as:

```go
if `isDCCFuncCalled` ...
```

Not sure - asking / raising as it was not immediately obvious what the two variables were doing.

Reply:
Or even use `has` or `is`. /shrug

Reply:
using the whole name instead of acronym  would be more readable

Reply:
The function it's called starts with 'is' to indicated a true/false scenario for the actual code.

isDeploymentControllerCurrent?  Yes, it is (true), not it's not (false).

isDCCResult = our contrived function's return value.  I agree, the naming is not the best, but in the production code I think it reads cleanly; isDCCResult is only used here in testing.

Reply:
> isDCCResult = our contrived function's return value. I agree, the naming is not the best, but in the production code I think it reads cleanly; isDCCResult is only used here in testing.

I wouldn't make a distinction between "production" and test code. Both are production, IMO.